tests/recon/torch/modules/test_latent.py

```diff
@@ -0,0 +1,60 @@
+import torch
+import unittest
+from abc import ABC, abstractmethod
+from typing import Iterator
+import torch.nn as nn
+from bdpy.recon.torch.modules import latent as latent_module
+
+
+class DummyLatent(latent_module.BaseLatent):
+    def __init__(self):
+        self.latent = torch.tensor([1.0])
```

Comment:
Latentクラスは学習可能なパラメーターをインスタンス変数で保持する実装を想定しているので、以下の実装に変更したいです

```suggestion
        self.latent = nn.Parameter(torch.tensor([1.0]))
```